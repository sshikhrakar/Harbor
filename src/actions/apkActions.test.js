import {
  installApk,
  installApkErrored,
  installApkFulfilled,
} from './apkActions';

describe('ACTIONS: APK', () => {

  it('should create an action that triggers apk install', () => {
    expect(
      installApk('/sdcard/com/testharbor/something.apk')
    ).toMatchSnapshot();
  });

  it('should create a fulfilled action for apk install', () => {
    expect(installApkFulfilled()).toMatchSnapshot();
  });

  it('should create a errored action for apk install', () => {
    expect(installApkErrored()).toMatchSnapshot();
  });


});
