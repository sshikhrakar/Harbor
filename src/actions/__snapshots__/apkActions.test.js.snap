// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ACTIONS: APK should create a errored action for apk install 1`] = `
Object {
  "type": "APK/INSTALL_APK_ERRORED",
}
`;

exports[`ACTIONS: APK should create a fulfilled action for apk install 1`] = `
Object {
  "type": "APK/INSTALL_APK_FULFILLED",
}
`;

exports[`ACTIONS: APK should create an action that triggers apk install 1`] = `
Object {
  "payload": Object {
    "path": "/sdcard/com/testharbor/something.apk",
  },
  "type": "APK/INSTALL_APK",
}
`;
